src/components/ArticleCard/index.js

```diff
@@ -1,9 +1,56 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
+import { PropTypes } from 'prop-types';
+import './ArticleCard.scss';
+import Icon from '../Icon';
 
-const ArticleCard = () => (
-  <div data-test="articleCardComponent">
-    <h1>Greetings from the commandos</h1>
-  </div>
-);
+const ArticleCard = (props) => {
+  const { type, data } = props;
+  return (
+    <div className={`${type}`}>
+      <div className="image-container">
+        <img className="img" alt="logo" src={data.image} />
+      </div>
+      <div className="article-details">
+        <div className="title-container">
+          <label className="title">{data.title}</label>
+          <label className="author-name">{data.authorName}</label>
+          <label className="description">{data.description}</label>
+        </div>
+        <div className="icons">
+          <div className="like">
+            <Icon name="likes" className="like" />
+            <label className="icon-label">{data.likes}</label>
+          </div>
+          <div className="dislike">
+            <Icon name="dislikes" />
+            <label className="icon-label">{data.dislikes}</label>
+          </div>
+          <div className="comment">
+            <Icon name="comments" />
+            <label className="icon-label">{data.comments}</label>
+          </div>
+          <label className="hor-readTime">{`${data.readTime} min read`}</label>
+        </div>
+        <label className="ver-readTime">{`${data.readTime} min read`}</label>
+      </div>
+    </div>
+  );
+};
+
+ArticleCard.propTypes = {
+  type: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/require-default-props
+  data: PropTypes.shape({
+    image: PropTypes.string,
+    title: PropTypes.string.isRequired,
+    authorName: PropTypes.string.isRequired,
+    description: PropTypes.string,
+    likes: PropTypes.number,
+    dislikes: PropTypes.number,
+    comments: PropTypes.number,
+    readTime: PropTypes.number,
+  }),
+};
 
```

Comment:
Nice Job monday, but I dont see the default propTypes Object in this file and others Please can you add that

Reply:
I don't think its necessary but I have added it in case I missed something.